src/coverlet.core/Helpers/FileSystem.cs

```diff
@@ -3,11 +3,46 @@
 
 namespace Coverlet.Core.Helpers
 {
-    internal class FileSystem : IFileSystem
+    public class FileSystem : IFileSystem
     {
-        public bool Exists(string path)
+        public virtual bool Exists(string path)
```

Comment:
please add a comment for `void` i.e. "We need to partial mock this class on tests"

Reply:
Done